src/components/ec-phone-number-input/ec-phone-number-input.vue

```diff
@@ -158,6 +158,8 @@ import { computed, ref } from 'vue';
 
 import type { Maybe, ZIndexLevel } from '../../../global';
 import useConfig from '../../composables/use-ec-config';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Do we need these ignores?

Reply:
Yes because since directives are not use in the `<script>` we are getting a TS error that said we must import them as types. If we do that directives doesn't work, so in order to continue using them we have to ignore TS in those imports.